src/gtest/test_libzendoo.cpp

```diff
@@ -625,6 +626,71 @@ TEST(SidechainsField, NakedZendooFeatures_EmptyTreeCommitmentCalculation)
     EXPECT_TRUE(scTxCommitmentHash == emptySha) <<scTxCommitmentHash.ToString() << "\n" << emptySha.ToString();
 }
 
+/**
+ * @brief This test checks the computation of the commitment tree of a known block.
+ * The block is read from a raw byte string stored in a file.
+ * This same check is performed on the SDK side to spot any possible difference.
+ * 
+ * Please, see the unit test blockWithCertificateWithCustomFieldAndBitvectorMixedScVersions
+ * on the SDK side for further details and the Python test sc_getscgenesisinfo.py for the
+ * generation of the test block.
+ * 
+ * https://github.com/HorizenOfficial/Sidechains-SDK/commit/460ad1f0984e96925e20be8e6c434d2acda439d4
+ */
+TEST(SidechainsField, CommitmentComputationFromSerializedBlock)
+{
+    SelectParams(CBaseChainParams::REGTEST);
+    const BlockchainTestManager& testManager = BlockchainTestManager::GetInstance();
+
+    // Read hex string from file
+    std::ifstream t("./src/gtest/test-data/block_with_v0_and_v1_certificates.hex");
```

Comment:
There might be problems if the UT is not executed from the standard path. In such case the resulting blockData would not be valid because the ifstream would not be valid too

Reply:
Ok, seems fine!